modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -711,6 +711,13 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 		contact.setSuffixListTypeId();
 	}
 
+	public void method(
+		DDMFormViewFormInstanceRecordsDisplayContext
+			ddmFormViewFormInstanceRecordsDisplayContext){
+		
+		ddmFormViewFormInstanceRecordsDisplayContext.getSearchContainer();
+	}
```

Comment:
@MicaelleSilvaa this breakline was the SF or you?

Reply:
This breakline was the SF.

Reply:
@MicaelleSilvaa okay then, Do a rebase, fix the conflicts locally, and then resend the changes. Once the changes pass the tests again the PR is approved and you will be free to proceed with forwarding.